test/setup/iam-projects/iam.tf

```diff
@@ -62,8 +62,8 @@ locals {
     "roles/dlp.inspectTemplatesEditor",
     "roles/dlp.user",
     "roles/serviceusage.serviceUsageAdmin",
-    "roles/cloudkms.cryptoKeyEncrypter",
     "roles/iam.serviceAccountTokenCreator",
+    "roles/cloudkms.cryptoOperator",
```

Comment:
is this only for the automated build tests?

Reply:
We are keeping this only for testing purposes in the example/standalone, otherwise the build will break. We should remove the `cloudkms.cryptoOperator` role in the next days. I'm already working on a solution for it.

Reply:
We changed the `wrapped_key.sh` that is running in the `examples/standalone` to add the role `cloudkms.cryptoOperator` and remove after the usage.